brevbaker-api-model-common/src/main/kotlin/no/nav/pensjon/brev/api/model/maler/LetterRequest.kt

```diff
@@ -21,4 +21,6 @@ data class BestillRedigertBrevRequest<T : Brevkode<T>>(
     val felles: Felles,
     val language: LanguageCode,
     val letterMarkup: LetterMarkup,
-)
```

Comment:
Hmm, skribenten krever at letterData for redigerbare brev er `RedigerbarBrevdata`. Burde vi ha det samme kravet her slik at vi er konsistent?

Reply:
Hmm, ja, det gir jo ikkje meining at det ikkje skal vera det her.

Reply:
Meh, det trekkjer med seg litt deserialisering, så var ikkje så rett fram som ein kunne tru. Lager ei trello-oppgåve på det.